src/components/Collection/CollectionBlock.tsx

```diff
@@ -28,7 +28,9 @@ export const CollectionBlock = props => {
           onClick={handleWishlist}
         />
         <Link to={`/collections/cabinets/${collect.id}/${collect.slug}`}>
```

Comment:
Change `collect` to `collection`

Either use 1 letter: `c` for short lambda functions like `collections.map(c => c.namae)` or use the full name to keep naming consistent.